spec/docs/accounts.rb

```diff
@@ -0,0 +1,37 @@
+module Docs
+  module Accounts
+    extend Dox::DSL::Syntax
+
+    # define common resource data for each action
+    document :api do
+      resource "Accounts" do
+        endpoint "/accounts"
+        group "Accounts"
+      end
+
+      group "Accounts" do
+        desc "The Accounts API is only accessible for super admins. A super admin account can manage other accounts."
+      end
+    end
```

Comment:
Interesting, how the doc will look?

Reply:
https://scfm.docs.apiary.io/#reference/accounts/accounts/create-an-account